documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -9,9 +9,18 @@
 This procedure describes how to generate a reassignment JSON file.
 Use the reassignment file with the `kafka-reassign-partitions.sh` tool to reassign partitions after scaling a Kafka cluster.
 
+You run the tool from an interactive pod container connected to the Kafka cluster.
+
 The steps describe a secure reassignment process that uses mTLS.
 You'll need a Kafka cluster that uses TLS encryption and mTLS authentication.
 
+You'll need the following to establish a connection:
+
+* The cluster CA certificate and password generated by the Cluster Operator when the Kafka cluster is created.
+* The user CA certificate and password generated by the User Operator when a user is created for client access to the Kafka cluster
+
+In this procedure, the CA certificates and passwords are extracted from the cluster and user secrets that contain them in PKCS #12 (`.p12` and `.password`) format.
```

Comment:
when reading "password" ... it was not clear to me which one the statement was referring to tbh.
I had to scroll and see the kubectl command to get what it meant.
We are talking about the "password" of the PKCS12 store containing the CA certificates.
Can we try to clarify here?

Reply:
I've added the following

> The passwords allow access to the `.p12` files that contain the certificates.
> You use the `.p12` files to specify a truststore and keystore to authenticate connection to the Kafka cluster.

Reply:
I am fine with the note which clarifies but can we say even "...the CA certificates and corresponding stores passwords are extracted ...". In addition to the note I mean. But if it doesn't sound ok for you no need to change.